doc/src/fix_nvt_sllod.txt

```diff
@@ -177,3 +182,7 @@ Same as "fix nvt"_fix_nh.html, except tchain = 1.
 
 :link(Daivis)
 [(Daivis and Todd)] Daivis and Todd, J Chem Phys, 124, 194103 (2006).
+
+:link(Daivis2)
```

Comment:
@sjplimp please note, that `:link(XXX)` entries have to be unique across **all** `*.txt` files. The multiple uses of the same link causes the translation to html via sphinx to not work correctly. there should be a warning printed about this, when you run `make html`.